.github/workflows/grimoirelab-release.yml

```diff
@@ -298,6 +298,39 @@ jobs:
         #TODO: Change to the latest version once the new release is created
         run: pip install git+https://github.com/Bitergia/release-tools.git#egg=release-tools
 
+      - id: semverup
+        name: Update version number
+        run: |
+          # Check what version to update: patch, minor or major
```

Comment:
I think the way to obtain the change of version lloks like magic :).

I think a better approach would be to have bitwise comparisons. For example, if the results from `changed-version` jobs return 4 (100), 2 (010), or 1 (001) and you consider each bit like a change in the major, minor, or patch, you can so something like `$(( ${{ needs.grimoirelab-toolkit.outputs.changed_version }}|${{ needs.grimoirelab-perceval.outputs.changed_version }}| ... | ${{ needs.grimoirelab-graal.outputs.changed_version }})` that will give you a number in between 7 and 0 with the bites activated. Then if you do and move bits from left or write you can know if what changed was a major, minor, or patch, you can also do the same with an `AND` operation (&) comparing with 4,2, or 1. These values can stored into variables/const, so they can have a better meaning.

Think about it like if you were comparing masks or flags. It's something very typical and common, which makes more sense than compare 3, 2, or 1.

What do you think? Does it make sense?

Reply:
I also think we should define variables like BUMP_MAJOR=4, etc, at the beginning of every job, so a reader can understand better what's going on and these numbers are not magic numbers.